### PR TITLE
Adding `Pyload-ng` , `flask-themes2` and `bitmath`

### PR DESCRIPTION
**Pull request by HeavyTony2**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by HeavyTony2** [comment]
@conda-forge/help-python
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=502895&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=06d62fd2-ab49-5add-9ee5-83b6882d7d2b&l=6915

I tested this on my PC and it does work on Windows.  https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=502895&view=artifacts&pathAsName=false&type=publishedArtifacts

File: recipes/pyload-ng/meta.yaml
Comment (carterbox) [inline recipes/pyload-ng/meta.yaml:2]: This is a pre-release, but there is a stable release available. Could you publish the latest stable version instead?

Once you have a feedstock, you can publish prereleases in a pre-release channel. https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=pre%20release#pre-release-builds

File: recipes/bitmath/meta.yaml
Comment (carterbox) [inline recipes/bitmath/meta.yaml:24]: I feel that there is a missing dependency on a `progressbar` called progress bar. https://github.com/tbielawa/bitmath/blob/1.3.3.1/requirements-py3.txt

or a run_constrained? Is it optional or not?

**Review by carterbox — CHANGES_REQUESTED** [review]

File: recipes/pyload-ng/meta.yaml
Comment (HeavyTony2) [reply recipes/pyload-ng/meta.yaml:2]: I think I'm better wait for a stable release on Python3. I'm not able to futurize the current stable one. https://github.com/pyload/pyload/blob/stable/module/setup.py

File: recipes/bitmath/meta.yaml
Comment (HeavyTony2) [reply recipes/bitmath/meta.yaml:24]: Technically, this is optional.
https://github.com/tbielawa/bitmath/blob/19a06eceb464c83899352bc9ebfe2d5e0b9ffa22/bitmath/integrations/__init__.py#L33

However, I'm tempted to indeed add `progressbar33`, `click` and `argparse` to the run dependancies.  Those are integrations.

File: recipes/pyload-ng/meta.yaml
Comment (carterbox) [reply recipes/pyload-ng/meta.yaml:2]: OK. Are you going to add a `conda_build_config.yaml` that points this recipe to the dev channel or do you want this PR to wait until they release stable for python 3?

File: recipes/bitmath/meta.yaml
Comment (carterbox) [reply recipes/bitmath/meta.yaml:24]: If optional it's fine, just remember to add run_constrained if they ever have version requirements.

**Review by carterbox — CHANGES_REQUESTED** [review]
Please see comment about dev channel for pyload-ng.

File: recipes/pyload-ng/meta.yaml
Comment (HeavyTony2) [reply recipes/pyload-ng/meta.yaml:2]: @carterbox Is it possible to set the package to the dev channel without having a main one?

File: recipes/bitmath/meta.yaml
Comment (HeavyTony2) [reply recipes/bitmath/meta.yaml:24]: Didn't find any trace of versions requirements. Thanks.

File: recipes/pyload-ng/meta.yaml
Comment (carterbox) [reply recipes/pyload-ng/meta.yaml:2]: I guess we will find out when this feedstock is provisioned.

**Pull request merged**
